app/javascript/components/greeting.js

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import { useDispatch, useSelector } from "react-redux";
+import { fetchGreeting, selectAllMsg } from "../redux/reducers/messageSlice";
+
+const Greeting = () => {
+  const dispatch = useDispatch();
+  const { greetings } = useSelector(selectAllMsg);
+
+  console.log(greetings);
```

Comment:
OPTIONAL
- Well done using `console.log` for debugging. However, kindly don't forget to remove it from the code before pushing changes. 🙏 It makes the code less clean and reduces the optimization of the app.